sam_bot_description/launch/display.launch.py

```diff
@@ -40,6 +40,8 @@ def generate_launch_description():
          parameters=[os.path.join(pkg_share, 'config/ekf.yaml'), {'use_sim_time': LaunchConfiguration('use_sim_time')}]
     )
     
+    world_path=os.path.join(pkg_share, 'world/my_world.sdf'),
```

Comment:
Ideally, move this up to L10-ish with the other similar variable setting on pkg paths